src/MiniProfiler.Providers.PostgreSql/PostgreSqlStorage.cs

```diff
@@ -0,0 +1,489 @@
+﻿using Dapper;
+using Npgsql;
+using StackExchange.Profiling.Internal;
+using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace StackExchange.Profiling.Storage
+{
+    /// <summary>
+    /// Understands how to store a <see cref="MiniProfiler"/> to a PostgreSQL Server database.
+    /// </summary>
+    public class PostgreSqlStorage : DatabaseStorageBase
+    {
+        /// <summary>
+        /// The schema that the tables are stored in. 
+        /// </summary>
+        public readonly string Schema = null;
```

Comment:
The pattern here is to pull in the table names *with schema* (check out MySql and SqlServer) - this simplifies everything down to the same constructor as the other providers and makes maintenance easier and full control explicit. The tables don't even *have* to be in the same schema. Or any schema. It can be relative to the current user.

In short: we should remove this and offer the same constructor patterns as the existing providers here.